### ISSUE TITLE
Accessibility: Merge Sections inactive sections contrast

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#638]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Merge Sections

Issue notes:
The inactive section names in the right side "Review before merge" list  and some of the text on the Search by Course Name section have poor contrast (2.1:1; should be at least 4.5:1) The inactive Add and Undo buttons have a contrast of only 1.7:1.

Criterion:
1.4.3 Contrast (Minimum)
